modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -4720,6 +4720,83 @@ else if (objectField.compareBusinessType(
 						280, entry.getValue(), objectField.getObjectFieldId(),
 						objectField.getI18nObjectFieldName());
 				}
+
+				if (!objectField.hasUniqueValues()) {
+					continue;
+				}
+
+				long objectEntriesCount = 0;
+				Table<?> table = null;
+
+				try {
+					table = _objectFieldLocalService.getTable(
+						objectDefinition.getObjectDefinitionId(),
+						objectField.getName());
+
+					Column<?, Object> column =
+						(Column<?, Object>)table.getColumn(
+							objectField.getDBColumnName());
+
+					DynamicObjectDefinitionLocalizationTable
+						dynamicObjectDefinitionLocalizationTable =
+							(DynamicObjectDefinitionLocalizationTable)table;
+
+					JoinStep joinStep = DSLQueryFactoryUtil.countDistinct(
```

Comment:
Is there a way to call `ObjectEntryLocalServiceImpl#getObjectEntriesCount` method here?
The idea is to avoid this duplication of code. 
Maybe you can also use `_validateUniqueValueConstraintViolation` method.